src/MinterGateway.sol

```diff
@@ -401,22 +401,20 @@ contract MinterGateway is IMinterGateway, ContinuousIndexing, ERC712Extended {
     function deactivateMinter(address minter_) external onlyActiveMinter(minter_) returns (uint240 inactiveOwedM_) {
```

Comment:
why deactivateMinter is changed?

Reply:
Because many internal functions were removed, we can just do `_imposePenaltyIfMissedCollateralUpdates` instead of manual principal-based math, and let the deactivation run its course by converting all active to inactive. It costs a bit more, but way simpler, especially if it allows for the deleting of code.